common/djangoapps/util/enterprise_api.py

```diff
@@ -0,0 +1,62 @@
+"""
+Utilities to communicate with the Enterprise service.
```

Comment:
Should I just throw the API client class into the `enterprise_helpers` module, then? It's small and simple enough where it probably doesn't matter, and I'd consider the class to be an "Enterprise helper". ;)